docs/reference/rollup/apis/rollup-api.asciidoc

```diff
@@ -44,7 +43,7 @@ POST /my-index-000001/_rollup
 [[rollup-api-request]]
 ==== {api-request-title}
 
-`PUT /<index>/_rollup`
+`PUT /<index>/_rollup/<rollup_index>`
```

Comment:
nit: Maybe this should be `PUT /<index>/_rollup/<rollup-index>`
use a `-` instead of `_` for consistency with the rest of the docs (I was looking at [roll-over docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-rollover-index.html))

Reply:
Good call. I've fixed this up with efd2db0775d44152b05b4f113f2f5c95752cfed4.